core/src/avm2/events.rs

```diff
@@ -87,6 +87,9 @@ impl KeyModifiers {
 #[collect(no_drop)]
 pub enum EventData<'gc> {
     Empty,
+    Error {
```

Comment:
`ErrorEvent` subclasses `TextEvent`, so `EventData::Error` should have all the fields in `EventData::Text`.